src/views/Splash.vue

```diff
@@ -64,6 +64,7 @@
             role="button"
             draggable="false"
             to="/fr"
+            lang="fr"
```

Comment:
Github doesn't let me comment on the specific area (line 40 of this file) but you're missing a lang="en" for the English button.

Reply:
Added lang=en to line ~40 and 103. Good catch!